lib/board.rb

```diff
@@ -0,0 +1,52 @@
+
+class Board
+  attr_accessor :cells
+    def initialize(cells = [" ", " "," "," "," "," "," "," "," "])
+      @cells = cells
+    end
+
+    def reset!
+      @cells = [" ", " "," "," "," "," "," "," "," "]
```

Comment:
Another way to instantiate an Array with `i` values is `Array.new(9, " ")` (see [Array.new](http://ruby-doc.org/core-2.2.0/Array.html#method-c-new)), where `9` is the number of elements in the array and `" "` is the default value for the elements. It's more concise and will prevent errors from repeatedly typing `" "`.